state-sync/src/coordinator.rs

```diff
@@ -870,7 +870,9 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
         let network_sender = self
             .network_senders
             .get_mut(&peer.network_id())
-            .expect("missing network sender");
+            .ok_or_else(|| {
+                Error::UnexpectedError(format!("Missing network sender for peer: {:?}", peer))
+            })?;
```

Comment:
This is one that I'd keep.

Basically, this is saying you're expecting to have a network interface, and it's now not there.  It's not actually missing a `network sender` for the `peer` but for the `network_id`'s interface instead.